core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -746,7 +746,7 @@ class ContainerPool(
 
     // Getter/setter for this are above.
     private var _logDir = "/logs"
-    private val actionContainerPrefix = "wsk"
+    private val actionContainerPrefix = s"wsk${invokerInstance.toInt}_"
```

Comment:
wsk{instnaceid}_{containercounter} could make name below cases
wsk1_1
wsk11_1
wsk111_1
if we use wsk{invokerInstanceId} only, another container can be effected.